homework1/nullElements/nullElements.c

```diff
@@ -0,0 +1,51 @@
+﻿#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <time.h>
+
+// Функция подсчёта нулей в массиве.
+// На вход принимает указатель на массив и его размер.
+// Возвращает целое число - количество нулевых элементов.
+int nullCounter(int* arrayForCounting, size_t arraySize) {
+    int counter = 0;
+
+    for (size_t i = 0; i < arraySize; ++i) {
+        if (!arrayForCounting[i])
```

Comment:
```suggestion
        if (arrayForCounting[i] == 0)
```
Это то же самое, но явно сравнивает с нулём, а не использует неочевидные правила преобразования типов